doc/news/changes/minor/20190411TimoHeister

```diff
@@ -0,0 +1,3 @@
+If present, we will detect the LLD linker ld.lld at configuration time.
```

Comment:
Would you mind prefixing the first sentence with something like `New:` like we usually do?